HelpMyStreetFE/HelpMyStreetFE/Views/Pages/Terms.cshtml

```diff
@@ -17,7 +17,7 @@
                 <li>Anyone in the UK who is over 18, has a valid passport or driving licence and is able to verify their identity with our partner, Yoti, can register as a volunteer on HelpMyStreet.org.</li>
                 <li>Anyone with a genuine need and currently living in the UK can request help, although we may not be able to match you with a volunteer until there are enough volunteers in your area. If this is the case, we&rsquo;ll let know before you submit any personal details.</li>
                 <li>Anyone can request help on behalf of someone who has a genuine need and is currently living in the UK.</li>
-                <li>Everyone who uses the site must support the aims of the community-led support organisations and <a href="http://HelpMyStreet.org">org</a>, which is to support those in need in the UK.</li>
+                <li>Everyone who uses the site must support the aims of the community-led support organisations and HelpMyStreet.<a href="http://HelpMyStreet.org">org</a>, which is to support those in need in the UK.</li>
```

Comment:
Let's include HelpMyStreet in the hyperlink.  (I know that's not precisely how it looks on the Word doc.)